source/common/protobuf/utility.cc

```diff
@@ -482,7 +482,8 @@ void MessageUtil::loadFromFile(const std::string& path, Protobuf::Message& messa
     }
     return;
   }
-  if (absl::EndsWith(path, FileExtensions::get().Yaml)) {
+  if (absl::EndsWith(path, FileExtensions::get().Yaml) ||
+      absl::EndsWith(path, FileExtensions::get().Yml)) {
```

Comment:
maybe using `absl::EndsWithIgnoreCase` is better, feel loose check better than strict check here. But yes, this isn't the scope of this PR also.

Reply:
Yeah this is a good idea. I will do this and a few more tests per @htuch